molgenis-data-postgresql/src/main/java/org/molgenis/data/postgresql/PostgreSqlQueryGenerator.java

```diff
@@ -967,8 +967,11 @@ private static String getSqlDropTable(String tableName) {
               Boolean bool = (Boolean) postgreSqlVal;
               //noinspection ConstantConditions (getPostgreSqlQueryValue() != null if r.getValue()
               // != null)
-              if (bool) predicate.append(" IS TRUE");
-              else predicate.append(" IS FALSE");
+              if (Boolean.TRUE.equals(bool)) {
+                predicate.append(" IS TRUE");
+              } else {
+                predicate.append(" IS FALSE");
```

Comment:
bool== null will now result in predicate addition ' IS FALSE'. Is this the desired behavior?